yellowbrick/gridsearch/base.py

```diff
@@ -0,0 +1,59 @@
+"""
+Base class for grid search visualizers
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import numpy as np
+from ..utils import is_gridsearch
+from ..base import ModelVisualizer
+from ..exceptions import YellowbrickTypeError
+
+
+##########################################################################
+## Base Grid Search Visualizer
+##########################################################################
+
+class GridSearchVisualizer(ModelVisualizer):
```

Comment:
Very good set up of this into the Yellowbrick class hierarchy!

Reply:
👍